packages/server/server.ts

```diff
@@ -34,10 +34,13 @@ tracer.init({
 tracer.use('ioredis').use('http').use('pg')
 
 process.on('SIGTERM', async (signal) => {
+  const RECONNECT_WINDOW = process.env.WEB_SERVER_RECONNECT_WINDOW
+    ? parseInt(process.env.WEB_SERVER_RECONNECT_WINDOW, 10) * 1000
+    : 60_000 // ms
```

Comment:
-1 We should move this top level. That way it's evaluated already on first importing the file and a `NUMBER_FORMAT_EXCEPTION` would interrupt startup. It's better than finding invalid environment variables only at runtime.

Reply:
Just at the top of the file is fine.